patches/tModLoader/Terraria/ModLoader/ModLoader.cs

```diff
@@ -31,6 +31,7 @@ public static class ModLoader
 		public static string LastLaunchedTModLoaderAlphaSha;
 		public static bool ShowWhatsNew;
 		public static bool AlphaWelcomed;
+		public static bool SeenMigrateTo143BranchMessage;
```

Comment:
We want to show it on _every_ launch for people who aren't on the branch. GoG players don't matter, but you should use the result from `SteamApps.GetCurrentBetaName` to control the visibility of the message